src/test/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/ReadFilterPluginUnitTest.java

```diff
@@ -1,30 +1,20 @@
-package org.broadinstitute.hellbender.cmdline.GATKPlugin;
+package org.broadinstitute.hellbender.engine.filters;
```

Comment:
Change to its package in GATK to keep the same organization. Do it in the last commit to keep the changes for easy review.